### PR TITLE
Update Dodge-the-Creeps to latest Godot version

### PR DESCRIPTION
**Pull request by Bromeon**
Updates the Dodge-the-Creeps example to version https://github.com/godotengine/godot/commit/4935493f720b0c4e925b0643960b5949bedae0c7.

Tested on Windows.

### CODE REVIEW

File: 2d/dodge_the_creeps/HUD.gd [hunk 1]
```diff
@@ -23,7 +23,7 @@ func update_score(score):
 
 func _on_StartButton_pressed():
 	$StartButton.hide()
-	emit_signal("start_game")
+	var _ignore = emit_signal("start_game")
```
Comment (adamscott) [inline 2d/dodge_the_creeps/HUD.gd:26]: godotengine/godot#69002 was just merged! You can safely remove `var _ignore` variable declarations as there will not be any warning (unless the user change its project settings).

**Comment by adamscott** [comment]
Can you update the [documentation code](https://github.com/godotengine/godot-docs), while being at it?

**Comment by Bromeon** [comment]
Updated the example to remove the `var _ignore = ...` workaround.

> Can you update the [documentation code](https://github.com/godotengine/godot-docs), while being at it?

Since I don't have that repo forked and don't know what exactly would need updating, I'd appreciate if someone who knows could do that directly 😉

**Comment by adamscott** [comment]
Here. The GDScript of this directory inside the `.rst` files. 

https://github.com/godotengine/godot-docs/tree/master/getting_started/first_2d_game

**Comment by Bromeon** [comment]
My GDScript changes are limited to two renamings:
* `offset` -> `progress`
* `RigidDynamicBody2D` -> `RigidBody2D`

I saw the first uses this in the docs:
```gdscript
mob_spawn_location.progress_ratio = randi()
```
is that correct? Should we not use `progress_ratio = randf()`?

**Comment by akien-mga** [comment]
Yes it should be `progress_ratio = randf()`.

**Comment by akien-mga** [comment]
Might have been superseded by #782, should check which changes may still be relevant.

**Pull request closed without merging**